src/app/ngrx/reducers/filter.reducer.ts

```diff
@@ -18,7 +18,6 @@ export interface FilterModel {
   podcastSeriesId?: number;
   page?: number;
   standardRange?: string;
-  range?: any[];
```

Comment:
stray range that shouldve been removed with my last review that took out the prev and next buttons